group_vars/researchdata/production.yml

```diff
@@ -1,16 +1,28 @@
-db_host: "{{ vault_postgres_host }}"
+---
+db_host: "lib-postgres-prod3.princeton.edu"
 db_clusteradmin_password: "{{ vault_postgres_admin_password }}"
-db_clusteradmin_user: "{{ vault_postgres_admin_user }}"
+db_clusteradmin_user: "postgres"
+
+desired_nodejs_version: "v8.17.0"
 
 postgres_host: "{{ db_host }}"
 postgres_port: "5432"
+pg_hba_postgresql_user: "all"
```

Comment:
It strange that this needs to be copied everywhere: https://github.com/pulibrary/princeton_ansible/search?q=pg_hba_postgresql_user   I would have expected the defaults to be part of the role.

Reply:
there will be instances where on wants to restrict more but I think I over stated its importance.

Reply:
https://github.com/pulibrary/princeton_ansible/issues/2870